stdlib/socket.pyi

```diff
@@ -438,6 +438,37 @@ if sys.platform == "win32":
         SIO_LOOPBACK_FAST_PATH as SIO_LOOPBACK_FAST_PATH,
         SIO_RCVALL as SIO_RCVALL,
     )
+if sys.version_info >= (3, 12):
+    from _socket import (
+        IP_ADD_SOURCE_MEMBERSHIP as IP_ADD_SOURCE_MEMBERSHIP,
+        IP_BLOCK_SOURCE as IP_BLOCK_SOURCE,
+        IP_DROP_SOURCE_MEMBERSHIP as IP_DROP_SOURCE_MEMBERSHIP,
+        IP_PKTINFO as IP_PKTINFO,
+        IP_UNBLOCK_SOURCE as IP_UNBLOCK_SOURCE,
+    )
+
+    if sys.platform == "win32":
+        from _socket import (
+            AF_HYPERV as AF_HYPERV,
```

Comment:
In `socket.pyi`, unlike in `_socket.pyi`, this is a member of the `AddressFamily` enum:

```pycon
>>> import socket
>>> socket.AF_HYPERV
<AddressFamily.AF_HYPERV: 34>
```